features/finders.feature

```diff
@@ -69,6 +69,16 @@ Feature: Filtering documents
     And I can see that Google won't index the page
     Then I browse to a huge page number and get an appropriate error
 
+  @javascript
+  Scenario: Updating search results
+    Given a finder with paginated results exists
+    And I visit the benefits-reform page
+    Then I should see results and pagination
+    And I fill in a keyword that should match no results
+    Then I should not see results and pagination
+    And I click the xxxxxxxxxxxxxxYYYYYYYYYYYxxxxxxxxxxxxxxx remove control
```

Comment:
This requires internal knowledge of the step definition for `And I fill in a keyword that should match no results`.

```suggestion
    When I click the remove control for the unmatchable keyword
```

Reply:
It does, yes, but it also reuses an existing method for clicking a given keyword facet tag. I think I'd rather reuse code here.